src/plone/volto/summary.py

```diff
@@ -7,4 +7,5 @@ class JSONSummarySerializerMetadata:
     def default_metadata_fields(self):
         return {
             "image_field",
+            "image_scales",
```

Comment:
@ericof what will happen if a backend does not have an image_scales field in the catalog metadata? Will it break?

Reply:
Looking at https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/serializer/summary.py I think it will either break or add "image_scales" to the serialization even of items that don't have any scales. Would be best to have a test.

Reply:
I will work on a proper test for this, but from my mind, I do recall the value is returned as null

Reply:
That sounds likely, because of this converter which turns the Missing.Value from the catalog brain into None: https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/serializer/converters.py#L200-L203

Reply:
Tests added